mobile/v1/redshift/sql-runner/sql/standard/01-base/01-main/03-sessions-to-process.sql

```diff
@@ -0,0 +1,71 @@
+/*
+   Copyright 2020-2021 Snowplow Analytics Ltd. All rights reserved.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
```

Comment:
I wonder if it's simpler to wrap the feature switch around the entire statement? 

Another thing to consider is whether the two can be in separate files, and we let the playbooks do the work of choosing what to run.

No strong stance here just wondering if the code can be more readable.

Reply:
Yeh I agree it can get a bit cluttered. I prefer the splitting it into two statements rather than files. I will see how it improves readability.